docs/docs/manuals/user-passwordless.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Passwordless
+---
+
+## Register Passwordless Authentication
+
+Add passwordless authentication to secure your account.
+
+1. Go to personal information - passwordless authentication
+2. Click add authenticator
+3. Enter a name for your authentication
+4. You have three options to register a new method
+   - Directly register a new method
+   - Send a link to your email address
+   - Generate a QR Code to scan with another device (e.g. Mobile Phone)
+5. Directly register a new method
+   a. Your device/browser will show you the possibilities you have
+   b. Choose your preferred method
+   c. Verify your method (e.g. Finger Scan, Face Recognition, Hardware Token, etc..)
+
+:::caution
+If you use different browsers, make sure you register all of them, otherwise you will not be able to use passwordless registration everywhere
```

Comment:
```suggestion
If you use different browsers, make sure you register all of them, otherwise you will not be able to use passwordless registration everywhere.
```